app/src/main/java/mn/devfest/sessions/details/SessionDetailsFragment.java

```diff
@@ -193,10 +211,22 @@ private void displaySpeakers(@Nullable ArrayList<String> speakers) {
 
         //Add SpeakerViews to the Speaker Layout
         for (String speakerId : speakers) {
-            Speaker speaker = mDataSource.getSpeakerById(speakerId);
-            SpeakerView speakerView = new SpeakerView(getActivity());
-            speakerView.setSpeaker(speaker, false);
-            mSpeakerLayout.addView(speakerView);
+            mFirebaseDatabaseReference.child(DEVFEST_2017_KEY).child(SPEAKERS_CHILD_KEY)
```

Comment:
Same deal. It would be better to put this in another class so the fragment doesn't specify as much of the details of using Firebase.